jerry-core/parser/js/js-scanner.c

```diff
@@ -3214,6 +3214,14 @@ scanner_scan_all (parser_context_t *context_p, /**< context */
   PARSER_TRY_END
```

Comment:
Thewre is a code above:
```
#if ENABLED (JERRY_ES2015)
    context_p->status_flags &= (uint32_t) ~PARSER_IS_GENERATOR_FUNCTION;
#endif /* ENABLED (JERRY_ES2015) */
```
Looks unnecessary.

Reply:
Looks but this removal not corresponds to the fix.